### PR TITLE
feat/sqlalchemy key

### PR DESCRIPTION
**Pull request by rellerreller**

### CODE REVIEW

**Comment by rellerreller** [comment]
-1 do not merge this until feat/sqlalchemy-secretdata has been merged.

**Comment by rellerreller** [comment]
This is back up for review now that the dependent patch has been merged.

File: kmip/tests/unit/pie/objects/test_symmetric_key.py [hunk 1]
```diff
@@ -317,9 +322,296 @@ def test_not_equal_on_type_mismatch(self):
...
+        session = Session()
+        session.add(key)
+        session.commit()
+
+        session = Session()
+        get_obj = session.query(SymmetricKey).filter(
+            ManagedObject.unique_identifier == key.unique_identifier
+            ).one()
+        session.commit()
+        print("get_obj.value = %s" % get_obj.value)
```
Comment (rellerreller) [inline kmip/tests/unit/pie/objects/test_symmetric_key.py]: Remove this debug print statement.

**Comment by PeterHamilton** [comment]
:+1: Looks good to me!

**Pull request merged**